public/scripts/suggestion.js

```diff
@@ -0,0 +1,17 @@
+$( document ).ready(function() {
```

Comment:
Since we use `(function(){` notation in every js files in place of document.ready, please make the necessary change for the sake of consistency